libs/state/actions/src/lib/actions.factory.ts

```diff
@@ -69,24 +71,28 @@ export class RxActionFactory<T extends Partial<Actions>> implements OnDestroy {
    *
    */
   create<U extends ActionTransforms<T> = {}>(transforms?: U): RxActions<T, U> {
-    const subjects: SubjectMap<T> = {} as SubjectMap<T>;
-    this.subjects.push(subjects);
+    const subjectMap: SubjectMap<T> = {} as SubjectMap<T>;
+    const effectMap: EffectMap<T> = {} as EffectMap<T>;
```

Comment:
Would it be more convenient and performant to use a real `Map` instead?

Reply:
yes, probably!

Reply:
Good point. I would say in the old code it is irrelevant in the new it should be considered.